drivers/rmnet/perf/Android.mk

```diff
@@ -20,10 +19,6 @@ ifeq ($(call is-board-platform-in-list, msmnile),true)
 LOCAL_C_INCLUDES := $(LOCAL_PATH)/../../../../../../../kernel/msm-4.14/include/
 endif #End of check for msmnile include
 
-ifeq ($(call is-board-platform-in-list, kona),true)
-LOCAL_C_INCLUDES := $(LOCAL_PATH)/../../../../../../../kernel/msm-4.19/include/
-endif #End of check for kona include
-
 RMNET_PERF_BLD_DIR := ../../vendor/qcom/opensource/data-kernel/drivers/rmnet/perf
 DLKM_DIR := ./device/qcom/common/dlkm
```

Comment:
> Can you clarify this? As @jerpelea mentioned in [sonyxperiadev/kernel-techpack-audio#42 (comment)](https://github.com/sonyxperiadev/kernel-techpack-audio/pull/42#issuecomment-1110667733) (but strangely just merged this PR thinking thinkingthinkingconfused) perhaps we should fix the configuration - or maybe nothing is wrong after all. Maybe our `edo` platform is underperforming or misconfigured because these not being fixed, but most likely we're not using any android binaries from these _kernel_ modules after all - I don't remember.

For example here, referencing a non-existing folder and causing a build failure. I am not sure if all of them need to be removed, maybe keep some of them will give a better performance etc. but at least doing this will give the same result when either using `KONA = kona` or `KONA = sm8250`.

Reply:
sorry, I think not this one, let me find the one I was talking about.